src/redux/categories/Categories.js

```diff
@@ -0,0 +1,34 @@
+const CHECK_STATUS = 'CHECK STATUS';
```

Comment:
- [ ] It would be better if our action types follow the `ducks` pattern. For example; `const CHECK_STATUS = bookstore/books/CHECK_STATUS` or in your case `const CHECK_STATUS = "bookstore/books/CHECK_STATUS"`